.landscaper/installation.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Installation
+metadata:
+  name: gardener-extension-provider-aws
+  namespace: garden
+spec:
+  componentDescriptor:
+    ref:
+      repositoryContext:
+        type: ociRegistry
+        baseUrl: eu.gcr.io/sap-gcp-cp-k8s-stable-hub/examples/landscaper/temp
```

Comment:
What's this?

Reply:
We have used the installation for local testing. We have removed it.